dspace/config/registries/dspace-types.xml

```diff
@@ -0,0 +1,20 @@
+<dspace-dc-types>
+
+    <dspace-header>
+        <title>DSpace Types</title>
```

Comment:
Tiny thing, but I'd recommend we name this something like "DSpace Internal Types Registry" just to make it clear these are internal fields.  Plus that aligns well with the titles in other registries.